build/openwisp_nginx/Dockerfile

```diff
@@ -21,8 +21,14 @@ ENV MODULE_NAME=nginx \
     NGINX_ADMIN_ALLOW_NETWORK=all \
     NGINX_HTTPS_ALLOWED_IPS=all \
     NGINX_HTTP_ALLOW=False \
-    NGINX_SSL_CONFIG='gzip off;' \
     NGINX_SERVER_NAME_HASH_BUCKET=32 \
+    NGINX_SSL_CONFIG='' \
+    NGINX_80_CONFIG='' \
+    NGINX_GZIP_SWITCH=off \
+    NGINX_GZIP_LEVEL=6 \
+    NGINX_GZIP_PROXIED=any \
+    NGINX_GZIP_MIN_LENGTH=1000 \
+    NGINX_GZIP_TYPES=* \
```

Comment:
To explicitly set gzip_types do you configure a quoted string of types (ex. NGINX_GZIP_TYPES="text/plain image/svg+xml application/json")?

Reply:
No, quotes are not required in the `.env` file! :smile: